admin_guide/assembly_restoring-cluster.adoc

```diff
@@ -0,0 +1,66 @@
+[[admin-guide-restore]]
+= Restoring your cluster
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:context: admin-restore-cluster
+
+toc::[]
+
+== Overview
+
+In {product-title}, you can _restore_, recreating state from separate storage,
+at the cluster level.
```

Comment:
Restore what?

Reply:
I take it that this topic works on multiple levels? You can restore an entire cluster or each of the listed components (master, nodes, etcd, OCP sevices, projects, applications, PVCs)? 
If I restore the cluster as a whole, do I always need to restore the components?

Reply:
I think it depends on what you've backed up and what you've broken. :)

I'm going with "In {product-title}, you can _restore_ your cluster and its components by
recreating cluster elements, including nodes and applications, from separate storage."